jest.config.js

```diff
@@ -1,3 +1,5 @@
+process.env.TZ = 'UTC';
```

Comment:
I guess this is deliberate, but feels very strange.

Reply:
Yeah, had the same feeling but apparently this is the way jest wants the time zone to be set. I have not found any other way to get the timezone "mocked".